### PR TITLE
added solution.

### PR DESCRIPTION
**Pull request by Merkald**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,60 @@
  */
 
 public class Plane {
+    private int size;
+    private String model;
+    private String number;
+    private long price;
+    private int age;
 
+    private Plane(PlaneBuilder pb) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: try to create builder, set few fields and then create Plane using build(). Past that code as answer for this comment(don't push it)

Comment (Merkald) [reply src/main/java/core/basesyntax/Plane.java]: Plane.PlaneBuilder builder = null;
        builder.setprice(5);
        builder.setage(4);
        Plane plane =builder.build();

Comment (Merkald) [reply src/main/java/core/basesyntax/Plane.java]: Try to build in main mehod and take java.lang.NullPointerException. Not understand reason(

Comment (Merkald) [reply src/main/java/core/basesyntax/Plane.java]: Пробував знайти рішення в інтернеті, ніде не знайшов структури за вимогами завдання(або конструктори пусті або вони публічні)

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,60 @@
...
+        }
+
+        public PlaneBuilder setprice(long price) {
+            this.price1 = price;
+
+            return this;
+        }
+
+        public Plane build() {
+            return Plane.this;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Plane.java]: what this method should return?

File: src/main/java/core/basesyntax/Plane.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/Plane.java]: look closely to your code, you assigned builder to null and then trying to invoke some method on null which is always will give you NullPointerException

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/core/basesyntax/Plane.java (hunk 2 above)
Comment (Merkald) [reply src/main/java/core/basesyntax/Plane.java]: return new Plane(this);

**Pull request closed without merging**